src/main.css

```diff
@@ -508,6 +511,7 @@ input::file-selector-button {
 
 	/* a-specific resets */
 	color: var(--fg);
+  cursor: default;
```

Comment:
@dz4k I just realized that setting `cursor: default;` here overrides the `[aria-disabled="true"], [disabled] {cursor: not-allowed}` rule in `sanitize.css`.

The line was added so as to give a default cursor on `a.\<button\>`, instead of the pointer that it was inheriting from `a`. Should I create a separate rule just for this one property, or is there some way to avoid over-specifying the value given in `sanitize.css`?

Reply:
Fixed this by scoping the cursor property to `:is(a)` in following commit.